numba/unicode.py

```diff
@@ -815,6 +818,30 @@ def unicode_strip_types_check(chars):
         raise TypingError('The arg must be a UnicodeType or None')
 
 
+@register_jitable
+def _handle_capital_sigma(s, l, idx):
+    #This algorithm is an approximate translation of:
```

Comment:
Suggest to specify what function do and returns here using """ """ doc-string

Reply:
I will do it.